printing/ppextend.ml

```diff
@@ -75,12 +75,13 @@ type generic_notation_printing_rules = {
 }
 
 let generic_notation_printing_rules =
-  Summary.ref ~name:"generic-notation-printing-rules" (NotationMap.empty : generic_notation_printing_rules NotationMap.t)
+  Summary.ref ~stage:Synterp ~name:"generic-notation-printing-rules" (NotationMap.empty : generic_notation_printing_rules NotationMap.t)
```

Comment:
alternatively maybe this should be set by the Interp object instead of the Synterp object?

Reply:
It looks ok this way, cheaper than adding an interp phase to `Reserved Notation`